cmd/kpod/common.go

```diff
@@ -50,30 +78,59 @@ func getStore(c *cli.Context) (storage.Store, error) {
 	return store, nil
 }
 
+func getCopyOptions(reportWriter io.Writer, signaturePolicyPath string, srcDockerRegistry, destDockerRegistry *dockerRegistryOptions, signing signingOptions) *cp.Options {
+	if srcDockerRegistry == nil {
+		srcDockerRegistry = &dockerRegistryOptions{}
+	}
+	if destDockerRegistry == nil {
+		destDockerRegistry = &dockerRegistryOptions{}
+	}
+	srcContext := srcDockerRegistry.getSystemContext(signaturePolicyPath)
+	destContext := destDockerRegistry.getSystemContext(signaturePolicyPath)
+	return &cp.Options{
+		RemoveSignatures: signing.RemoveSignatures,
+		SignBy:           signing.SignBy,
+		ReportWriter:     reportWriter,
+		SourceCtx:        srcContext,
+		DestinationCtx:   destContext,
+	}
+}
+
+func getPolicyContext() (*signature.PolicyContext, error) {
+	var systemContext types.SystemContext
+	policy, err := signature.DefaultPolicy(&systemContext)
+	if err != nil {
+		return &signature.PolicyContext{}, err
+	}
+	return signature.NewPolicyContext(policy)
+}
+
 func findImage(store storage.Store, image string) (*storage.Image, error) {
 	var img *storage.Image
 	ref, err := is.Transport.ParseStoreReference(store, image)
 	if err == nil {
-		img, err = is.Transport.GetStoreImage(store, ref)
+		img, err := is.Transport.GetStoreImage(store, ref)
+		if err != nil {
+			return nil, err
+		}
+		return img, nil
 	}
-	if err != nil {
-		img2, err2 := store.Image(image)
-		if err2 != nil {
-			if ref == nil {
-				return nil, errors.Wrapf(err, "error parsing reference to image %q", image)
-			}
-			return nil, errors.Wrapf(err, "unable to locate image %q", image)
+	img2, err2 := store.Image(image)
+	if err2 != nil {
+		if ref == nil {
+			return nil, errors.Wrapf(err, "error parsing reference to image %q", image)
 		}
-		img = img2
+		return nil, errors.Wrapf(err, "unable to locate image %q", image)
 	}
+	img = img2
 	return img, nil
 }
 
-func getCopyOptions(reportWriter io.Writer) *cp.Options {
+/*func getCopyOptions(reportWriter io.Writer) *cp.Options {
```

Comment:
Yes, I commented it out to see if something would break and forgot to remove it.  Nice catch